librocksdb_sys/src/lib.rs

```diff
@@ -510,6 +512,7 @@ macro_rules! ffi_try {
 // TODO audit the use of boolean arguments, b/c I think they need to be u8
 // instead...
 extern "C" {
+    pub fn crocksdb_resume(db: *mut DBInstance, err: *mut *mut c_char);
```

Comment:
placed in the same order as c.h?

Reply:
yeah, but i found there is no order in extern "C" code block. like `crocksdb_status_ptr_get_error`, `crocksdb_resume` is placed under below `crocksdb_status_ptr_get_error` in c.h. anyway just don't mind that